docs/release/release-tasks.md

```diff
@@ -169,6 +170,17 @@ This can be done by:
 2. If nobody is working on an issue in the milestone, drop it from the milestone.
 3. Ensuring we have a plan to get `release-blocking` issues implemented in time.
 
+#### [Continuously] Bump the Go version
```

Comment:
This is primarily to keep the patch version up to date, right? If so, can we make that explicit that this item is only to update the patch version and not the minor version.

The minor version is updated during Kubernetes bump and not here, isn't it?

Reply:
In most cases patch

Sometimes

> Note: If the Go minor version of one of our supported branches goes out of supported, we should consider bumping 
to a newer Go minor version according to our [backport policy](./../../CONTRIBUTING.md#backporting-a-patch).

Reply:
Kubernetes bump never bumps the go version on release branches as that part of the work is only done on main